tests/Operation/AggregateFunctionalTest.php

```diff
@@ -110,20 +110,23 @@ public function testTypeMapOption(array $typeMap = null, array $expectedDocument
         $this->assertEquals($expectedDocuments, $results);
     }
 
-    public function testTypeMapInline()
+    /**
+     * @dataProvider provideTypeMapOptionsAndExpectedDocuments
+     */
+    public function testTypeMapOptionWithoutCursor(array $typeMap = null, array $expectedDocuments)
     {
         if (version_compare($this->getServerVersion(), '3.6.0', '>=')) {
-            $this->markTestSkipped('Inline aggregations are not supported');
+            $this->markTestSkipped('Aggregations with useCursor == false are not supported');
         }
 
         $this->createFixtures(3);
 
-        $pipeline = [['$match' => ['x' => 1]], ['$out' => $this->getCollectionName() . '.output']];
+        $pipeline = [['$match' => ['_id' => ['$ne' => 2]]]];
 
-        $operation = new Aggregate($this->getDatabaseName(), $this->getCollectionName(), $pipeline, ['typeMap' => ['root' => 'array'], 'useCursor' => false]);
+        $operation = new Aggregate($this->getDatabaseName(), $this->getCollectionName(), $pipeline, ['typeMap' => $typeMap, 'useCursor' => false]);
         $results = $operation->execute($this->getPrimaryServer());
 
-        $this->assertTrue($results instanceof \MongoDB\Model\TypeMapArrayIterator);
+        $this->assertInstanceOf(\ArrayIterator::class, $results);
```

Comment:
Ah, good idea to use [`::class`](http://php.net/manual/en/language.oop5.basic.php#language.oop5.basic.class.class) here. We couldn't use it before since we were supporting 5.4, but that's no longer a concern.

I would suggest `use ArrayIterator` up top and removing the `\` here, just so we're consistent about listing all classes at the top of each file (e.g. `stdClass`).